service/test/ServiceIOGroupTest.cpp

```diff
@@ -37,6 +37,7 @@
 #include "geopm_topo.h"
 
 #include "geopm/Helper.hpp"
+#include "geopm/PlatformIO.hpp"
```

Comment:
Why do we need this header for, there are no other changes to the file which would indicate need to include this header in the file.